README.md

```diff
@@ -35,7 +37,7 @@ Ansible role for a bigbluebutton installation (following the documentation on ht
 | `bbb_secret` | define the secret for bbb | `none` | `set this if you want to define the bbb-conf -secret. Otherwise the secret is generated by bbb`  
 | `bbb_cpuschedule` | CPUSchedulingPolicy | `true` | Disable to fix [FreeSWITCH SETSCHEDULER error][bbb_cpuschedule] |
 | `bbb_freeswitch_ipv6` | Enable IPv6 support in FreeSWITCH | `true` | Disable to fix [FreeSWITCH IPv6 error][bbb_freeswitch_ipv6] |
-| `bbb_freeswitch_external_ip` | Set stun server for sip and rtp on FreeSWITCH | `stun:{{ (bbb_stun_servers | first).server }}` | WARNING: the value of the default freeswitch installation is `stun:stun.freeswitch.org` |
+| `bbb_freeswitch_external_ip` | Set stun server for sip and rtp on FreeSWITCH | `stun:{{ (bbb_stun_servers \| first).server }}` | WARNING: the value of the default freeswitch installation is `stun:stun.freeswitch.org` |
```

Comment:
Please remove this slash

Reply:
Its unrelated to the PR but without the slash the | is interpreted as table next column switch. I assume that's the reason for this change. But I assume having the slash will make the default value look strange so maybe (independently from this PR) write the default value differently.

Reply:
<img width="2033" alt="image" src="https://user-images.githubusercontent.com/115045/98337373-d167cc80-2008-11eb-9a97-ebab9eac8a2b.png">
 on the left with my change, on the right how it currently is…

Reply:
but yeah it's not really scope of this PR so I'll remove it  so we can merge this

Reply:
Oh, sorry. I haven't noticed, that the table is broken. Well, i'll merge your PR now. But thanks for noticing...